pkg/shell/superuser.jsx

```diff
@@ -143,10 +143,23 @@ class LockDialog extends React.Component {
 }
 
 export class SuperuserDialogs extends React.Component {
-    constructor() {
+    constructor(props) {
         super();
```

Comment:
If you accept props, they should be propagated to the parent constructor.

Reply:
Good point, we don't need props in the constructor.  But also see: https://stackoverflow.com/questions/30571875/whats-the-difference-between-super-and-superprops-in-react-when-using-e

Reply:
Interesting! I suppose I'm thinking too much in Python or C++ terms, where `this.props` must be set by the constructor. Nevermind then, I learned something new :)

Reply:
Yeah, I would understand it if one had to _always_ pass props to super(), but it seems to be optional, and then it must be always optional, no matter what args your own constructor takes.